src/app/base/components/DebounceSearchBox/DebounceSearchBox.tsx

```diff
@@ -55,7 +59,7 @@ const DebounceSearchBox = ({
             "debounce-search-box__spinner-container u-vertically-center",
             { "nudge-left": !!searchText }
           )}
-          data-testid="debouncing-spinner"
+          data-testid={TestIds.DebouncingSpinner}
```

Comment:
You could add an `aria-label` to the spinner icon instead and avoid having to use a test id.